angular/projects/common/src/app/entity/entity-add-edit/entity-add-edit.component.html

```diff
@@ -0,0 +1,82 @@
+<form [formGroup]="formGroup">
+  <!-- Add/Edit* -->
+  <div class="flex-container">
+    <div class="label-pair">
+      <label for="type">Entity Type</label>
+      <select name="type" id="type" formControlName="type" class="form-control">
+        <option (click)="updateUI()" *ngFor="let entityType of entityTypes" [ngValue]="entityType">
+          {{ entityType }}
+        </option>
+      </select>
+    </div>
+
+    <!-- Individual* -->
+    <!-- TODO enable/update as part of anonymous entity/document work. -->
+    <!-- <ng-container *ngIf="UIType === ENTITY_TYPE.Individual || UIType === ENTITY_TYPE.IndividualCombined">
+      <div class="label-pair">
+        <label for="anonymous">Anonymity</label>
+        <mat-slide-toggle
+          name="anonymous"
+          id="anonymous"
+          formControlName="anonymous"
+          (change)="updateAnonymousText($event.checked)"
+        >
+          {{ anonymousText }}
+        </mat-slide-toggle>
+      </div>
+    </ng-container> -->
```

Comment:
No commented code.

Reply:
So this is to support the anonymous stuff, which is up and coming 

> 
> 
> Also, we don't need separate commits for lint, just do them as part of your code cleanup before submitting. Please don't force push any edits to this PR any more as it breaks github tracking.

Some of the files linted arent related to this PR, but in theory would break the git actions that check linting.

I can put them in with the other commits, but then it would be misleading since no actual code was changed.

Reply:
This all has to do with the anonymous logic, which is related to issuedTo.

The only reason its commented out is because the dirty-fields issue breaks it, which I found out after writing all of it.

So I think it should stay, and either get un-commented when the dirty-fields PRs are all done OR this whole PR can wait for them to get merged, and I can un-comment it all at once as the last step of this PR.